test/testSamples/MaterialX/no-rule-studio-config-v1.0.0_aces-v1.3_ocio-v2.0.ocio

```diff
@@ -0,0 +1,1231 @@
+ocio_profile_version: 2
+
+environment:
+  {}
+search_path: ""
+strictparsing: true
+luma: [0.2126, 0.7152, 0.0722]
+name: studio-config-v1.0.0_aces-v1.3_ocio-v2.0
+description: |
+  Academy Color Encoding System - Studio Config [COLORSPACES v1.0.0] [ACES v1.3] [OCIO v2.0]
+  ------------------------------------------------------------------------------------------
+  
+  This "OpenColorIO" config is geared toward studios requiring a config that includes a wide variety of camera colorspaces, displays and looks.
+  
+  Generated with "OpenColorIO-Config-ACES" v1.0.0 on the 2022/10/26 at 05:59.
+
+roles:
+  aces_interchange: ACES2065-1
+  cie_xyz_d65_interchange: CIE-XYZ-D65
+  color_picking: sRGB - Texture
+  color_timing: ACEScct
+  compositing_log: ACEScct
+  data: Raw
+  matte_paint: sRGB - Texture
+  scene_linear: ACEScg
+  texture_paint: ACEScct
+
+file_rules:
+  - !<Rule> {name: Default, colorspace: ACES2065-1}
```

Comment:
No file rules. Means metadata will have to be used to get proper color correction.